contracts/Autopay.sol

```diff
@@ -216,14 +225,17 @@ contract Autopay is UsingTellor {
         _feed.interval = _interval;
         _feed.window = _window;
         _feed.priceThreshold = _priceThreshold;
+        _feed.feedsWithFundingIndex = 0;
```

Comment:
you don't have to initialize it to zero, can take this line out

Reply:
Removed